mlops/parallelm/mlops/channels/mlops_python_channel.py

```diff
@@ -27,9 +29,12 @@ def get_logger(self, name):
     def done(self):
         pass
 
-    def stat(self, name, data, model_id, category=None, model=None, model_stat=None):
+    def stat(self, name, data, model_id, category=None, model=None, model_stat=None, **kwargs):
```

Comment:
Why is the code for the confusion matrix in the python channel.. This should probably be in somewhere else.. thee channel should not handle the specific format of the stat.